### PR TITLE
contract: rename more functions

### PR DESCRIPTION
**Pull request by lucasmoura**
## Why is this needed?
We are renaming more contract module methods to align with their definition on the contract backend.

## Test Steps
Verify that no integration tests are failing due to this change

## Checklist

 - [x] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

## Does this PR require extra reviews?

 - [ ] Yes
 - [x] No

### CODE REVIEW

**Review by renanrodrigo — APPROVED** [review]

File: uaclient/contract.py [hunk 1]
```diff
@@ -16,21 +16,31 @@
...
-API_V1_TMPL_CONTEXT_MACHINE_TOKEN_RESOURCE = (
+API_V1_GET_CONTRACT_MACHINE = (
     "/v1/contracts/{contract}/context/machines/{machine}"
 )
-API_V1_RESOURCES = "/v1/resources"
-API_V1_TMPL_RESOURCE_MACHINE_ACCESS = (
+API_V1_UPDATE_CONTRACT_MACHINE = (
+    "/v1/contracts/{contract}/context/machines/{machine}"
+)
+API_V1_GET_AVAILABLE_RESOURCES = "/v1/resources"
```
Comment (alnvdl-work) [inline uaclient/contract.py:31]: Nitpick: it should have been named `GetAvailableResources`, but the official name on our side is just `AvailableResources`.

So I'd suggest renaming this to `API_V1_AVAILABLE_RESOURCES` to keep it fully aligned with the backend.

File: uaclient/contract.py [hunk 2]
```diff
@@ -16,21 +16,31 @@
...
 API_V1_UPDATE_ACTIVITY_TOKEN = (
     "/v1/contracts/{contract}/machine-activity/{machine}"
 )
-API_V1_CONTRACT_INFORMATION = "/v1/contract"
+API_V1_GET_CONTRACT_USING_TOKEN = "/v1/contract"
 
-API_V1_MAGIC_ATTACH = "/v1/magic-attach"
+API_V1_GET_MAGIC_ATTACH = "/v1/magic-attach"
+API_V1_POST_MAGIC_ATTACH = "/v1/magic-attach"
+API_V1_DELETE_MAGIC_ATTACH = "/v1/magic-attach"
```
Comment (alnvdl-work) [inline uaclient/contract.py:43]:
```
API_V1_GET_MAGIC_ATTACH_TOKEN_INFO
API_V1_NEW_MAGIC_ATTACH_TOKEN
API_V1_REVOKE_MAGIC_ATTACH_TOKEN
```

**Review by alnvdl-work — COMMENTED** [review]
Some suggestions to make it 100% aligned with ua-contracts.

**Comment by lucasmoura** [comment]
@alnvdl-work updated

File: uaclient/contract.py [hunk 3]
```diff
@@ -16,21 +16,31 @@
 from uaclient.defaults import ATTACH_FAIL_DATE_FORMAT
 from uaclient.entitlements.entitlement_status import UserFacingStatus
 
+# We are now describing each endpoint we use by the
+# type of HTTP request we are performing together with
+# endpoint name defined in the Contract backend. We must do
+# that to easily identify which endpoints are being used
+# by the Pro client
```
Comment (alnvdl-work) [inline uaclient/contract.py:23]:
```suggestion
# Here we describe every endpoint from the ua-contracts 
# service that is used by this client implementation.
```

**Review by alnvdl-work — APPROVED** [review]
LGTM. And thank you for this effort, this will really help us to debug and understand client behavior going forward!

**Comment by lucasmoura** [comment]
:facepalm: I just removed the extra file

**Pull request merged**